src/server/verification/processor/EnrollmentProcessor.js

```diff
@@ -140,6 +145,46 @@ class EnrollmentProcessor {
     }
   }
 
+  async checkExistence(enrollmentIdentifier, v1EnrollmentIdentifier) {
+    const [exists, v1Exists] = await Promise.all([
+      this.isIdentifierExists(enrollmentIdentifier),
+      v1EnrollmentIdentifier && this.isIdentifierExists(v1EnrollmentIdentifier)
+    ])
+
+    return { exists, v1Exists }
+  }
+
+  normalizeIdentifiers(enrollmentIdentifier, v1EnrollmentIdentifier = null) {
+    return {
+      identifier: enrollmentIdentifier.slice(0, 42),
+      v1Identifier: v1EnrollmentIdentifier ? v1EnrollmentIdentifier.replace('0x', '') : null
+    }
+  }
+
+  async verifyIdentifier(enrollmentIdentifier, gdAddress) {
+    // check v2, v2 identifier is expected to be the whole signature
+    if (enrollmentIdentifier.length < 42) {
+      return
+    }
+
+    const signer = recoverPublickey(
+      enrollmentIdentifier,
+      FV_IDENTIFIER_MSG2({ account: toChecksumAddress(gdAddress) }),
+      ''
+    )
+
+    if (!strcasecmp(signer, gdAddress)) {
+      throw new Error(`identifier signer doesn't match user ${signer} != ${gdAddress}`)
+    }
+  }
+
+  async getEnrollment(enrollmentIdentifier: string, customLogger = null): Promise<any> {
```

Comment:
revert this refactoring

Reply:
ok, but 
a) I will create a separate verification/utils/utils.js file
b) will move verifyIdentifier from verificationAPI there (as it will be also used from goodID middleware)
c) will keep normalizeIdentifiers, but move it also to utils.js as a simple function. Because this code part repeats in any FV endpoint, more that 5 times. And, as we agreed, if something repeats more than 2-3 times - we could refactor it by making a shared util